### ISSUE TITLE
How to launch the tabs example?

### ISSUE THREAD

**Issue by leoj3n** [matthewp/weston#2]
Currently I am in the root of the cloned repo, and doing:

```
http-server
```

Navigating in the browser to http://127.0.0.1:8080/examples/tabs/ gives a browser console error:

```
Error: Error loading "examples/tabs/tabs" at http://127.0.0.1:8080/examples/tabs/tabs.js
TypeError: Cannot read property 'parentNode' of undefined
	    at Object.replace (http://127.0.0.1:8080/node_modules/can-view-nodelist/can-view-nodelist.js:427:31)
	    at Object.live.list (http://127.0.0.1:8080/node_modules/can-view-live/lib/list.js:387:13)
	    at Text.callback (http://127.0.0.1:8080/lib/target.js:170:16)
	    at hydrateCallbacks (http://127.0.0.1:8080/node_modules/can-view-target/can-view-target.js:252:39)
	    at Object.hydrate (http://127.0.0.1:8080/node_modules/can-view-target/can-view-target.js:271:4)
	    at Function.eval [as renderer] (http://127.0.0.1:8080/lib/index.js:31:19)
	    at Constructor.setup (http://127.0.0.1:8080/node_modules/can-component/can-component.js:317:29)
	    at Function.newInstance (http://127.0.0.1:8080/node_modules/can-construct/can-construct.js:289:22)
	    at Constructor.init (http://127.0.0.1:8080/node_modules/can-construct/can-construct.js:637:29)
	    at new Constructor (eval at <anonymous> (http://127.0.0.1:8080/node_modules/can-construct/can-construct.js), <anonymous>:3:34)
	Evaluating http://127.0.0.1:8080/examples/tabs/tabs.js null
```

Also, there is this warning:

```
can-component.prototype.template: is deprecated and will be removed in a future release. Use can-component.prototype.view
```

The web page renders blank.

**Comment by matthewp** [comment]
Looks like something broke it. Not sure what at the present moment.  Probably a bug in a dependency.

**Comment by matthewp** [comment]
Will be fixed by https://github.com/matthewp/weston/pull/3

**Comment by matthewp** [comment]
Fixed in 0.2.1

**Comment by leoj3n** [comment]
Thanks, it's working now! 🍾
